src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -138,32 +142,25 @@ public Token login(String username, String password, String eauth)
      * @return object representing the scheduled job
      * @throws SaltStackException if anything goes wrong
      */
-    public Job startCommand(String target, String function, List<String> args,
+    public Job startCommand(final String target, final String function, List<String> args,
             Map<String, String> kwargs) throws SaltStackException {
-        // Setup lowstate data to send as JSON
-        JsonObject json = new JsonObject();
-        json.addProperty("tgt", target);
-        json.addProperty("fun", function);
-        // Non-keyword arguments
-        if (args != null) {
-            JsonArray argsArray = new JsonArray();
-            for (String arg : args) {
-                argsArray.add(new JsonPrimitive(arg));
+        Map<String, String> allKwargs = new LinkedHashMap() {
```

Comment:
Please add type parameters `<String, String>` to `new LinkedHashMap()` throughout this patch.